prisma/schema.prisma

```diff
@@ -82,6 +82,8 @@ model Votes {
   id                 Int              @id @default(autoincrement())
   user_id            String
   place_etiquette_id Int
+  experience_id      Int
```

Comment:
A Vote is related to a Place, not to an Experience. 

You are trying to work in "Votes" as they were "Helpfullness".